pkg/netpol/eval/resources.go

```diff
@@ -543,8 +600,15 @@ func (pe *PolicyEngine) AddPodByNameAndNamespace(name, ns string, nsLabels map[s
 		return nil, err
 	}
 	if pe.exposureAnalysisFlag && newPod.Name == k8s.RepresentativePodName { // if exposure-analysis and this is not a fake ingress-controller
+		// first compute a selector sorted (and unique) string to be used as a map key
+		// todo : the key of the map might also be the representativePeer.String() if fits (sorted and ensures equality)
+		// (when we decide how to implement it)?!
+		keyStrFromLabels := getSortedLabelsString(objLabels)
```

Comment:
The computation of  unique key from set  of labels already exists in a simpler version in `variantFromLabelsMap()` within `pkg\netpol\eval\internal\k8s\pod.go`.
Can remove implementation of `getSortedLabelsString`, and instead use this mechanism here as well.